### PR TITLE
Added testcase to validate 3422 timer while mme restarts

### PR DESCRIPTION
**Pull request by rsarwad**
[LTE] Added test case to validate the behavior of 3422 timer while MME restarts. 3422 timer is started after sending Detach Request message to receive Detach Accept message.

Description: To send Detach Request message, s1ap tester establishes dedicated bearer and then sends request to delete default bearer. Since delete bearer is initiated for default bearer, MME sends Detach Request to s1ap tester and starts 3422 timer. Then s1ap tester provides command to restart MME service.  After MME recovery, it is expected to send Detach Request again and s1ap responds with Detach Accept message.

### CODE REVIEW

File: lte/gateway/python/integ_tests/s1aptests/test_attach_nw_initiated_detach_with_mme_restart.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    def test_attach_nw_initiated_detach_with_mme_restart(self):
+        """
+        The test case validates retransmission of Detach Request after MME
+        restarts
+        Step 1: UE attaches to network
+        Step 2: Create dedicated bearer
+        Step 3: Send request to delete default bearer, since deletion is
+                invoked for default bearer, MME initaites detach procedure
+        Step 4: MME starts 3422 timer to receive Detach Accept message
+        Step 5: Send command to restart MME service to validate the behavior
```
Comment (ssanadhya) [inline lte/gateway/python/integ_tests/s1aptests/test_attach_nw_initiated_detach_with_mme_restart.py]:
```suggestion
        Step 5: Send command to restart MME service to validate the behavior of
```

File: lte/gateway/python/integ_tests/s1aptests/test_attach_nw_initiated_detach_with_mme_restart.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                3422 timer, on MME recovery, it sends Detach Request
+        Step 6: S1ap tester shall wait on Detach Request and send Detach Accept
+                message
+        """
+        num_ues = 1
+        self._s1ap_wrapper.configUEDevice(num_ues)
+
+        for i in range(num_ues):
```
Comment (ssanadhya) [inline lte/gateway/python/integ_tests/s1aptests/test_attach_nw_initiated_detach_with_mme_restart.py]: We don't need a for loop if it is just one ue.

**Review by ssanadhya — APPROVED** [review]
Minor comments, rest looks good.

File: lte/gateway/python/integ_tests/s1aptests/test_attach_nw_initiated_detach_with_mme_restart.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+        # Wait on EMM Information from MME
+        self._s1ap_wrapper._s1_util.receive_emm_info()
+
+        time.sleep(1)
+        print(
+            "********************** Adding dedicated bearer to IMSI",
+            "".join([str(i) for i in req.imsi]),
+        )
+        self._spgw_util.create_bearer(
```
Comment (rsarwad) [inline lte/gateway/python/integ_tests/s1aptests/test_attach_nw_initiated_detach_with_mme_restart.py]: Check the purpose of establishing dedicated bearer?

Comment (rsarwad) [reply lte/gateway/python/integ_tests/s1aptests/test_attach_nw_initiated_detach_with_mme_restart.py]: It's not required to send request to create dedicated bearer, so removed from test script

**Pull request merged**